_sass/uswds_overrides.scss

```diff
@@ -246,11 +246,11 @@ button,
 
   &.usa-button-secondary,
   &.usa-button-secondary:visited {
-    background-color: $color-bright;
+    background-color: white;
```

Comment:
should this be `$color-white` ?

Reply:
I used that because `$color-white` isn't included in `variables.scss`.

I now see that `$color-inverse` is `#fff`. That is not a good name. Happy to use that, tho, if you think it better.